src/odemis/acq/stream/_static.py

```diff
@@ -422,12 +422,12 @@ def dis_bbtl(v):
 
         # polarization VA
         # check if any polarization analyzer data, (None) == no analyzer data (pol)
-        if self._pos.keys()[0][-1]:
+        if list(self._pos.keys())[0][-1]:
             # use first entry in acquisition to populate VA (acq could have 1 or 6 pol pos)
-            self.polarization = model.VAEnumerated(self._pos.keys()[0][-1],
+            self.polarization = model.VAEnumerated(list(self._pos.keys())[0][-1],
                                 choices=polpositions)
 
-        if self._pos.keys()[0][-1]:
+        if list(self._pos.keys())[0][-1]:
```

Comment:
Drop that `if`, which is identical to the `if` just above, and merge the two blocks.